curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,434 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a common method for hiding content in HTML?
 
 #### --distractors--
 
-Placeholder distractor 1
+display: none;
 
 ---
 
-Placeholder distractor 2
+visibility: hidden;
 
 ---
 
-Placeholder distractor 3
+opacity: 0;
 
 #### --answer--
 
-Placeholder answer
+All of the above
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the minimum contrast ratio for normal text according to WCAG?
 
 #### --distractors--
 
-Placeholder distractor 1
+2:1
 
 ---
 
-Placeholder distractor 2
+3:1
 
 ---
 
-Placeholder distractor 3
+5:1
 
 #### --answer--
 
-Placeholder answer
+4.5:1
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool can help inspect element styles in a web browser?
 
 #### --distractors--
 
-Placeholder distractor 1
+Adobe Photoshop
 
 ---
 
-Placeholder distractor 2
+Microsoft Word
 
 ---
 
-Placeholder distractor 3
+Notepad
 
 #### --answer--
 
-Placeholder answer
+Chrome DevTools
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute is used to hide content from screen readers?
 
 #### --distractors--
 
-Placeholder distractor 1
+hidden
 
 ---
 
-Placeholder distractor 2
+display: none;
 
 ---
 
-Placeholder distractor 3
+visibility: hidden;
 
 #### --answer--
 
-Placeholder answer
+aria-hidden
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the ideal contrast ratio for large text?
```

Comment:
```suggestion
What is the minimum contrast ratio for large text?
```

I wouldn't say "ideal" here. WCAG is a bare minimum for accessibility. 3:1 is the bare minimum required for large text, not the "ideal".

Reply:
You have not addressed this @kalvakeerthi

Reply:
You still have not addressed this @kalvakeerthi please do so

Reply:
You still have not addressed this.